ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ovn/ipsec/azurefile-csi/provision/cucushift-installer-rehearse-azure-ipi-ovn-ipsec-azurefile-csi-provision-chain.yaml

```diff
@@ -0,0 +1,10 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-ovn-ipsec-azurefile-csi-provision
```

Comment:
I suggest to directly copy the steps of the chain to workflow yaml file

Reply:
Addressed, thanks for the suggestions!